plugins/package-managers/nuget/src/main/kotlin/utils/NuGetInspector.kt

```diff
@@ -197,6 +197,13 @@ private fun List<NuGetInspector.PackageData>.toPackageReferences(): Set<PackageR
         )
     }
 
+internal fun getIdentifierWithNamespace(name: String, version: String): Identifier {
```

Comment:
@mnonnenmacher am I correct to assume that `public` access to this would be required here in order for [`GenerateAspNetCoreCurationsTask` et al](https://github.com/oss-review-toolkit/ort-config/blob/80f72cbf3b6c0c3c9c62ef78b62fc9e2752bb427/tools/curations/buildSrc/src/main/kotlin/GenerateAspNetCoreCurationsTask.kt#L27) to make use of the new logic?

Reply:
If so, I'd probably move this to `NuGet.kt`, as strictly speaking the logic is not *nuget-inspector* specific.

Reply:
I think it would be better to make it public just in case there are changes in the future. Otherwise the copy might get out of sync.